tensorflow_datasets/scripts/refactoring/refactoring_dataset.py

```diff
@@ -0,0 +1,148 @@
+from absl import app
+from dataclasses import dataclass
```

Comment:
Could you format your code using https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style
If you're on window, make sure you have a recent version of pylint installed.

Reply:
Done